### PR TITLE
Feature/Complied Budget Application

### PR DESCRIPTION
**Pull request by Alibaba2023**
# Changes in the milestone:

- [x] ✅Testing requirements
- [x] ✅Create unit and integration tests for all the most important components of your RoR application.

Screenshot 2023-11-25 085905

- [x] ✅Technical requirements
- [x] ✅You should use Postgres as your database.
- [x] ✅You should use devise for authentication.
- [x] ✅You should validate all user input to make sure that anyone with bad intentions cannot compromise your app.
- [x] ✅You can use a view template engine of your choice (.erb, .slim, .haml).
- [x] ✅The project should be deployed.

Screenshot 2023-11-25 153459

- [x] ✅Your database schema should reflect the following structure:
- [x] ✅Video [link](https://drive.google.com/file/d/11W_0J-RmDL3RqkRky3wwz2DTmvTF-YCQ/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,291 +1,279 @@
...
 
-<!-- LIVE DEMO -->
+---
 
-## 🚀 Live Demo <a name="live-demo"></a>
+### 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [ ] video [link](https://drive.google.com/file/d/11W_0J-RmDL3RqkRky3wwz2DTmvTF-YCQ/view?usp=sharing)
+
```
Comment (zieeco) [inline README.md]: - Hey, as a requirement for this project, please, kindly deploy your app. currently you've not deployed your app.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Alibaba2023,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice project implementation ✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Alibaba2023) [reply README.md]: Hi @zieeco 
Thank you for the review.
But until now we haven't had any tutorial on how to deploy a Rails app and no one of my partners has deployed their projects.

File: README.md [hunk 2]
```diff
@@ -1,291 +1,269 @@
...
-
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🔭 Walk through video <a name="walk-through-video"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+<a href="https://drive.google.com/file/d/11W_0J-RmDL3RqkRky3wwz2DTmvTF-YCQ/view?usp=sharing">Video presentation of the Project</a>
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Munsa1) [inline README.md]: - [ ] As requested by the previous reviewer, kindly deploy your application as this is a core requirement. Here is a tutorial that may help [tutorial](https://www.youtube.com/watch?v=QJ4BcTQlKC0)


#### Feel free to ask for a second opinion from a senior reviewer if you are not satisfied with the review, but this is a requirement for this project

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Alibaba2023, 
required changes

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Alibaba2023, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**